content/en/security/explorer.md

```diff
@@ -26,17 +26,20 @@ From the [Security Signals Explorer][1], correlate and triage security signals.
 
 The Security Signals search results are displayed in the Security Signals Table.
 
-{{< img src="security/security_monitoring/explorer/security_signals_table.png" alt="The Security Signals table showing two account takeover signals" >}}
+{{< img src="security/security_monitoring/explorer/signal_panel_v2.png" alt="The Security Signals table showing two account takeover signals" >}}
```

Comment:
There's a vertical line that appears on the left side of the screenshot. Just FYI.